Rohit/PostgresAssignment/server.ts

```diff
@@ -0,0 +1,23 @@
+import { Context } from "vm";
+
+const koa = require("koa");
+const bodyParser = require("koa-bodyparser");
+const json = require("koa-json");
+const router = require('./router/router');
+
+const app = new koa();
+
+const port = 4001;
```

Comment:
you can read  port from config. ref https://github.com/adityajawade93/freshers-june2021/tree/main/jatin/postgres_assignment

Reply:
resolved